scripts/migrate.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 python manage.py runscript fix_migrations_for_kobocat_django_app
```

Comment:
As per our discussion, migrations must be fixed when upgrading KoboCAT migrations from `2.025.43` or `main` branch.